NBitcoin/Protocol/NetworkAddress.cs

```diff
@@ -5,113 +5,421 @@
 using System.Net;
 using System.Text;
 using System.Threading.Tasks;
+using NBitcoin.DataEncoders;
+using NBitcoin.BouncyCastle.Crypto.Digests;
+using NBitcoin.Crypto;
 
 namespace NBitcoin.Protocol
 {
 	public class NetworkAddress : IBitcoinSerializable
 	{
+		public const uint AddrV2Format = 0x20000000;

```

Comment:
```suggestion
                /// <seealso href="https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki"/>
		public const uint AddrV2Format = 0x20000000;
```

Possibly. V2 does not really make it clear what it is about.